os/configuration/dkms/index.md

```diff
@@ -6,5 +6,20 @@ layout: os-default
 
 ## DKMS
 
+DKMS is supported by running the DKMS scripts inside a container.  To compile any kernel modules you first need to download the kernel headers.  The kernel headers are available in the form of a system service.  Run the below to install the kernel headers:
 
-More information coming soon!
\ No newline at end of file
+```sh
+sudo ros service enable kernel-headers
+sudo ros service up -d kernel-headers
+```
+
+The above command will end up installing the kernel headers in `/lib/modules/$(uname -r)/build`.  This will make the kernel headers available to any container running in User Docker.  To install the kernel headers for use by container running in System Docker run the below commands:
+
+```sh
+sudo ros service enable kernel-headers-system-docker
+sudo ros service up -d kernel-headers-system-docker
+```
```

Comment:
Are all kernel headers (for user and system-docker) installed in  `/lib/modules/$(uname -r)/build`?